libpod/container.go

```diff
@@ -672,3 +691,11 @@ func (c *Container) isStopped() (bool, error) {
 	}
 	return c.state.State == ContainerStateStopped, nil
 }
+
+// Save container state to the database
+func (c *Container) Save() error {
```

Comment:
Don't think this needs to be exposed. We should never need to call this from outside libpod.

Reply:
Fixed